ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -349,13 +349,11 @@ private class SqlLedger(
       (archive, PackageDetails(archive.getPayload.size().toLong, knownSince, sourceDescription)))
     ledgerDao
       .uploadLfPackages(submissionId, packages)
-      .map {
-        case PersistenceResponse.Ok =>
-          UploadPackagesResult.Ok
-        case PersistenceResponse.Duplicate =>
-          // Note: package upload is idempotent, apart from the fact that we only keep
-          // the knownSince and sourceDescription of the first upload.
-          UploadPackagesResult.Ok
+      .map { _ =>
+        // Note: We discard any information about duplicates vs. non-duplicates here,
+        // as package upload is idempotent, apart from the fact that we only keep
+        // the knownSince and sourceDescription of the first upload.
+        UploadPackagesResult.Ok
```

Comment:
This makes the code less resistant against adding a new case to `PersistenceResponse` (e.g., `PersistenceResponse.Error`). I don't have a strong opinion, but my personal preference would be listing both cases explicitly.

Reply:
It's not strictly necessary right now, but since it's information that it's available to the data access object I see no reason to discard it upstream either. Furthermore, it would be very difficult to understand in which case to return `Ok` and in which return `Duplicate` if the end result is mixed. Do you think we should add the concept of `Duplicate` to this layer as well?